pkg/importer/http-datasource.go

```diff
@@ -248,7 +251,7 @@ func createHTTPReader(ctx context.Context, ep *url.URL, accessKey, secKey, certD
 
 	total, err := getContentLength(client, ep, accessKey, secKey)
 	if err != nil {
-		return nil, total, err
+		total = 0
```

Comment:
if an error happens total is already 0, so no need to explicitly set it here. But we do need to capture the head failed (so we can set brokenForQemu). We also need to communicate that back to the caller of this function, so we need an extra return parameter because below we can potentially find a content length.